app/javascript/src/dashboard.js

```diff
@@ -12,19 +12,6 @@ const defineCaseContactsTable = function () {
   )
 }
 
-const defineSupervisorsDataTable = function () {
-  $('table#supervisors').DataTable(
```

Comment:
I'm removing this piece of code, because it was causing conflict. More info [here](https://datatables.net/manual/tech-notes/3)

Reply:
Looks good. What about the 3 tests crashnig ?

Reply:
Thanks @efgalvao! I'm going to fix that, implementing that filter functionality soon. I think it's the only thing is missing